components/location/popup.jsx

```diff
@@ -0,0 +1,319 @@
+        href="${location.seekingMoneyURL}"
+        target="_blank"
+        // onclick="captureOutboundLink('${location.seekingMoneyURL}', 'donation')"
+      >
+        {getTranslation("seeking_money_link")}
+      </a>
+    );
+  }
+  return (
+    <span className="seekingMoney seeking-money card-badge">
+      <span data-translation-id="seeking_money">
+        {getTranslation("seeking_money")}
+      </span>{" "}
+      {link}
+    </span>
+  );
+};
+
+const OpenHours = ({ closingHours, id, openingHours }) => {
+  if (openingHours === "never" || openingHours === "not today") {
+    return <PopupSection content={openingHours} id={id} />;
+  } else {
+    const openHours = closingHours.map((closeHour, i) => {
+      <p>
+        <span>{openingHours[i]}</span> to <span>{closingHours[i]}</span>
+      </p>;
+    });
+    return <PopupSection content={openHours} id={id} />;
+  }
+};
+
+const NoIdNeeded = ({ location }) => {
+  const { getTranslation } = useContext(LanguageContext);
+
+  if (location.someInfoRequired === true) {
+    return (
+      <span
+        className="noIdNeeded card-badge"
+        // data-translation-id="some_info_required"
+      >
+        {getTranslation("some_info_required")}
+      </span>
+    );
+  } else {
+    return (
+      <span
+        className="noIdNeeded card-badge"
+        // data-translation-id="no_id_needed"
+      >
+        {getTranslation("no_id_needed")}
+      </span>
+    );
+  }
+};
+
+const WarmingSite = () => {
+  const { getTranslation } = useContext(LanguageContext);
+  return (
+    <span
+      // Kanad note: this translation ID is wrong!
+      // data-translation-id="warming_site"
+      className="warmingSite card-badge"
+    >
+      {getTranslation("warm_up_here")}
+    </span>
+  );
+};
+
+// const Address = ({ address }) => {
+//   const googleMapDirections = `https://maps.google.com?saddr=Current+Location&daddr=${encodeURI(
+//     address
+//   )}`;
+//   return (
+//     <address>
+//       <a
+//         href={googleMapDirections}
+//         target="_blank"
+//         onClick="captureOutboundLink('${googleMapDirections}', 'directions')"
+//         rel="noreferrer"
+//       >
+//         {address}
+//       </a>
+//     </address>
+//   );
+// };
+
+const PublicTransit = ({ location }) => {
+  const options = location.publicTransitOptions;
+  const routes = options.map((option, i) => (
+    <React.Fragment key={`${option.routeName}-${i}`}>
+      {/** Kanad Note: I had to change all these span tags to div tags
+       * since it was causing issues when doing a ReactDOMServer render.
+       *
+       * I also got rid of the separate div with alt text and
+       * just added it as an aria-label below.
+       * Source: https://stackoverflow.com/a/38438002 */}
+      <div
+        aria-label={option.altText}
+        className="transit-route-option"
+        role="text"
+      >
+        <i className="material-icons-round route-icon">{option.icon}</i>
+        <h5
```

Comment:
You mentioned this I think but this probably should not be an h tag - probably doesn't need to change right now but acknowledging